src/main/resources/interface-specification.yml

```diff
@@ -13,6 +13,8 @@ info:
 servers:
   - url: '/api-gateway/{{version}}'
     description: This service is the interoperability API gateway
```

Comment:
@beetlecrunch is this still useful?

Reply:
Does it have any impact on infra?

Reply:
@galales Do you know if it would be impactful?

Reply:
good point, @micdes-pagopa? 😬

Reply:
mic said that the `/api-gateway` shouldn't be present because he will inject it through APIGW